README.md

```diff
@@ -23,6 +23,7 @@ This project contains the following folders.
 | [adt-adx-queries](/adt-adx-queries) | Contains sample queries that can be run with the Azure Digital Twins query plugin for Azure Data Explorer (coming soon). |
 | [azure-functions](/azure-functions) | Contains sample Azure functions that can be used (as an example) with Azure Digital Twins for managing data ingress and twin updates. |
 | [ISS](/ISS) | Project to track the International Space Station with Azure Digital Twins and Azure Data Explorer |
+| [bulk-import](/ndjson-generator) | Contains helper code to generate ndjson files that can be used for bulk import |
```

Comment:
When I first read this, I thought this was a helper to convert some other data (e.g., output from ADT's QueryApi) into ndjson - but when I wen through the code, I realized it's for generating random data. Consider clarifying in this description.

Reply:
Renamed for clarity